build.sbt

```diff
@@ -268,7 +268,7 @@ lazy val scalahostSbt = project
     bintrayRepository := "maven", // sbtPlugin overrides this to sbt-plugins
     testQuick := {
       // runs tests for 2.11 only, avoiding the need to publish for 2.12
-      RunSbtCommand("; plz 2.11.8 publishLocal " +
+      RunSbtCommand(s"; plz $ScalaVersion publishLocal " +
```

Comment:
This line will fail if the main Scala version becomes 2.12, the scripted test depends on a 2.11 Scala version.